### ISSUE TITLE
SendGrid, MailChimp support

### ISSUE THREAD

**Issue by GQAdonis** [humhub/humhub-modules-mail#44]
It would be nice to have support for third party mail services like SendGrid and MailChimp, so email messages can be sent safely and have a greater chance of reaching the target without being blocked by SPAM blockers.

Is this something already in progress?  If not, I would be willing to contribute.
